octobot_trading/exchanges/parser/orders_parser_ccxt.py

```diff
@@ -0,0 +1,47 @@
+import octobot_trading.enums as enums
+import octobot_trading.exchanges.parser.orders_parser as orders_parser
+
+
+class CCXTOrdersParser(orders_parser.OrdersParser):
+    """
+    override CCXTOrdersParser if you add support for a CCXT exchange
+    dont use GenericCCXTOrdersParser as a base
+
+    parser usage:   parser = CCXTOrdersParser(exchange)
+                    orders = await parser.parse_orders(raw_orders)
+                    order = await parser.parse_order(raw_order)
+    """
+
+    TIMESTAMP_KEYS: list = [enums.ExchangeOrderCCXTColumns.TIMESTAMP.value]
```

Comment:
I don't understand why we need those keys in the orders parser. I don't why we need to have it in the generic orders parsers but here we should only try ccxt well documented uniformized format in my opinion.